pom.xml

```diff
@@ -173,12 +173,18 @@
             <artifactId>junit</artifactId>
             <version>4.13.2</version>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.hamcrest</groupId>
+                    <artifactId>hamcrest-core</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
-          <groupId>org.hamcrest</groupId>
-          <artifactId>hamcrest-core</artifactId>
```

Comment:
When I try to remove this and run: mvn dependency:analyze, I get this error: [ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:3.2.0:analyze (default-cli) on project antisamy: Cannot analyze dependencies: Trying to force use of dependencies which are not declared: [org.hamcrest:hamcrest-core]